### ISSUE TITLE
Open links in default browser, not Safari

### ISSUE THREAD

**Issue by frankgerhardt** [DagAgren/toot#440]
My default browser is Firefox. Please use my default browser for opening links. If that is not possible on iOS, then add a preference where I can set the desired browser.

**Comment by DagAgren** [comment]
There is no default browser setting in iOS, and I don't really want the maintenance burden of collecting and updating a list of all possible alternate browsers, so I don't think I can realistically do anything about this as things stand.

**Comment by frankgerhardt** [comment]
I agree, it's not Toot's responsibility if iOS has no default browser. However, I checked Tweetbot and it has it's own preference for the browser. They took the pain to add it. 

I have in Firefox my bookmarks, my logins/sessions/cookies, my private tab settings and anti-tracking config. In Safari I have nothing. 

If Toot opens a link in Safari, I get tracked, I don't have my logins, I won't bookmark anything. I have to open the URL in Firefox. If I'm lucky there is an action for "Open in" but I don't know because I'm not a heavy Safari user. Most likely I use the clipboard to copy the URL. What a pain. How many clicks!?

If I were able to copy the link from the toot directly I could save a few clicks but that does not work #434. But still >1 clicks instead of 1 click.

For me that means, opening links has such a bad usability that I consider this a non-existent feature. This is so bad that I'd rather go back to the Mastodon web interface. If opening a link does not work without pain, what benefit is there in an app!?
